src/platform/plugins/private/presentation_panel/public/panel_component/presentation_panel_error_internal.tsx

```diff
@@ -257,9 +257,9 @@ const styles = {
   }),
   outerWrapperStyles: css({ overflow: 'auto', display: 'flex' }),
   innerWrapperStyles: ({ euiTheme }: UseEuiTheme) =>
-    css({ padding: euiTheme.size.m, margin: 'auto' }),
+    css({ padding: euiTheme?.size.m, margin: 'auto' }),
```

Comment:
Why is `euiTheme` optional here? That shouldn't be necessary - it's not optional in any other component (for example, `src/platform/packages/private/kbn-split-button/src/split_button.tsx`)

Reply:
There's a shocking amount of layers of `useEuiTheme` and `css` calls throughout the React Embeddable test, and for some reason `useEuiTheme` doesn't need to be mocked for any component except this one. But then when you try to mock just `euiTheme.size` so the error panel works, it breaks a dizzying amount of other things.

I'll give it another pass to see if I can come up with a good way of getting this to work. Not sure why `split_button` using the same structure has no problem working in Jest.

Reply:
lmao I just had to wrap the whole test in `<EuiThemeProvider>`

Whatever man computers are ridiculous